src/nvim/msgpack_rpc/channel.c

```diff
@@ -251,13 +259,10 @@ Object channel_send_call(uint64_t id,
     return NIL;
```

Comment:
In fact, there's something weird going on here (Github bug). I think I annotated the correct `return` (the one within `if (frame.errored)`. And so I see in the diff comment at main conversation page. But, for some reason, it is placed in the other return when going into the code. Strange.

In any case, the important thing is we do agree there's a missing decref within `if (frame.errored)`.